### PR TITLE
Add Atomx support

### PR DESCRIPTION
**Pull request by erikdubbelboer**
## Type of change
- [X] New bidder adapter

## Description of change
Add [Atomx](https://www.atomx.com) bidder adapter.

- test parameters for validating bids:
```
{
  bidder: 'atomx',
  params: {
    id: 2
  }
}
```
- erik@atomx.com
- [X] official adapter submission

### CODE REVIEW

**Comment by aneuway2** [comment]
Hey @erikdubbelboer , just a heads up that you may want to also submit a PR to have your adaptor documented on prebid.org :-)

https://github.com/prebid/prebid.github.io/tree/master/dev-docs/bidders

**Comment by erikdubbelboer** [comment]
@aneuway2 I already had this ready but was waiting for this to be merged. Created the pull request now anyways: https://github.com/prebid/prebid.github.io/pull/201

File: src/adapters/atomx.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+      bid = JSON.parse(bid);
+    } catch (_error) {
+      bid = {
+        cpm: 0,
+        code: ''
+      };
+      utils.logError(_error);
+    }
+    if (bid.cpm > 0) {
+      bidObject = bidfactory.createBid(1);
```
Comment (jaiminpanchal27) [inline src/adapters/atomx.js]: pass bidrequest object as second argument to match bid request/response pair

File: src/adapters/atomx.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+      bidObject.cpm = bid.cpm * 1000;
+      if (bid.adm) {
+        bidObject.ad = bid.adm;
+      } else {
+        bidObject.adUrl = bid.url;
+      }
+      bidObject.width = bid.width;
+      bidObject.height = bid.height;
+    } else {
+      bidObject = bidfactory.createBid(2);
```
Comment (jaiminpanchal27) [inline src/adapters/atomx.js]: pass bidrequest object as second argument to match bid request/response pair

**Comment by erikdubbelboer** [comment]
@jaiminpanchal27 sorry, I should have paid better attention. The bugs have been fixed.

**Comment by jaiminpanchal27** [comment]
@erikdubbelboer I have created a test page and the creatives loaded on the page always navigates the parent page.  
Is the external script loaded by adapter responsible for this behavior ?
Please update with good creative.

**Comment by erikdubbelboer** [comment]
@jaiminpanchal27 my mistake, I should have given you a proper ID. Try:
```js
{
  bidder: 'atomx',
  params: { id: 7395 }
}
```

**Comment by mkendall07** [comment]
LGTM.

**Comment by jaiminpanchal27** [comment]
This is merged into master. Please submit a PR to the [docs repo](https://github.com/prebid/prebid.github.io) to add a file for your adapter to the [bidders directory](https://github.com/prebid/prebid.github.io/tree/master/dev-docs/bidders) so your adapter's params will appear on the [bidders page](http://prebid.org/dev-docs/bidders.html). Thank you for contributing

**Comment by erikdubbelboer** [comment]
@jaiminpanchal27 See https://github.com/prebid/prebid.github.io/pull/201

**Comment by erikdubbelboer** [comment]
@mkendall07 Is there any reason why Atomx isn't in the ["New adapter" list here](https://github.com/prebid/Prebid.js/releases/tag/0.21.0) but instead separately?

**Comment by mkendall07** [comment]
@erikdubbelboer 
Looks like it was just missed. Fixed it now. Thanks

**Pull request merged**